src/ZoneServer/World/Storage/PersonalStorage.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Melia.Shared.Tos.Const;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Items;
+using Melia.Zone.Network;
+using Melia.Zone.Database;
+using Melia.Zone.World.Actors.Characters.Components;
+
+namespace Melia.Zone.World.Storage
+{
+	/// <summary>
+	/// Personal storage of a single character.
+	/// </summary>
+	public class PersonalStorage : Storage
+	{
+		/// <summary>
+		/// Character that owns this personal storage.
+		/// </summary>
+		public Character Owner { get; private set; }
+
+		/// <summary>
+		/// Whether the owner is currently browsing this storage.
+		/// </summary>
+		public bool IsBrowsing { get; private set; }
+
+		/// <summary>
+		/// Constructor.
+		/// </summary>
+		/// <param name="owner"></param>
+		public PersonalStorage(Character owner) : base()
+		{
+			this.Owner = owner;
+			this.IsBrowsing = false;
```

Comment:
This assigment is unnecessary because the property is initialized to its default value (`false`) automatically.